### DISCUSSION TITLE
Les tenues FDO ne chargent pas

### DISCUSSION THREAD

**Discussion by Kedrihan** [SOZ-Faut-etre-Sub/SOZ-FiveM-Server#149] [Q&A] [answered]
Hello !
Petite interrogation, je tente de mettre mon perso en tenue FDO, mais les tenues ne chargent pas.

Exemple avec la tenue de service LSPD : 
![image](https://user-images.githubusercontent.com/20105314/228600928-696ab99b-8688-4dab-8e21-b0051095dd1b.png)

La ressource [clothes] est pourtant bien lancée, contenant toutes les textures des tenues FDO, donc je n'arrive pas à comprendre pourquoi ça ne les prend pas en compte.

Quelqu'un a une idée d'où ça peut venir svp ?

Merci d'avance !

**Comment by Laikker** [comment, marked as answer]
Hello,
Le streaming de vetements demande un clef FiveM payante, sauf si tu mets un nombre de joueur max sur ton serveur suffisament bas pour qu'il soit considéré comme du dev.
Met un nombre de joueur max à 4 dans ta config serveur et ca devrais être mieux

**Reply by Kedrihan** [reply]
Merci beaucoup ! J'avais complètement oublié cette règle !
